backend/app.py

```diff
@@ -1,5 +1,10 @@
 # -*- coding: utf-8 -*-
 """Create an application instance."""
 from grant.app import create_app
+from grant.blockchain.bootstrap import send_bootstrap_data
 
 app = create_app()
+
+@app.before_first_request
```

Comment:
This was the best thing I could find in flask for making sure the app has bootstrapped before running some initializer code. However, this waits until the first request to the backend, which means we can be waiting for a user to hit the site. Kind of annoying.

@dternyak any thoughts on how to make this better?

Reply:
This is a simple solution, but we could have an uptime robot simply poll the backend every 1 minute that would cause this to trigger at worst within 1 minute of startup.